### ISSUE TITLE
About converting the DCN layer in the onnx model is not supported

### ISSUE THREAD

**Issue by wcz13228607727** [Bin-ze/BEVFormer_segmentation_detection#20]
I've successfully converted the onnx model and the inference visualization is successful. 
But in the configuration I removed the DCN layer of mmcv, which caused the accuracy of the result to drop severely.
Does the author have the results of the model trained by removing the DCN layer?

**Comment by Bin-ze** [comment]
Sorry, I didn't modify the original network part, just support segmentation. Therefore you should try to use the model I provide as pre-trained weights to fine-tune the model without using DCN

**Comment by wcz13228607727** [comment]
好的，谢谢

**Comment by wcz13228607727** [comment]
> 对不起，我没有修改原来的网络部分，只是支持分段。因此，您应该尝试使用我提供的模型作为预训练权重来微调模型，而无需使用 DCN

ok，谢谢
